etest/tests.py

```diff
@@ -123,11 +123,11 @@ def run(self):
                 container=container,
             )
 
-            self.failed = is_interrupted or bool(docker.container.wait(container_name))
+            exit_code = docker.container.wait(container_name)["StatusCode"]
```

Comment:
Why give this a name for only one use?

Reply:
This is a debugging thing that managed to sneak in.

Reply:
Could just codify it with a debug logging statement. ;)